Zend/zend_timer.h

```diff
@@ -0,0 +1,29 @@
+/*
+   +----------------------------------------------------------------------+
+   | Copyright (c) The PHP Group                                          |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 3.01 of the PHP license,      |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | https://www.php.net/license/3_01.txt                                 |
+   | If you did not receive a copy of the PHP license and are unable to   |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@php.net so we can mail you a copy immediately.               |
+   +----------------------------------------------------------------------+
+   | Author: Kévin Dunglas <kevin@dunglas.dev>                            |
+   +----------------------------------------------------------------------+
+ */
+
+#ifndef ZEND_TIMER_H
+#define ZEND_TIMER_H
+
+# ifdef ZEND_TIMER
+
+#include "zend_long.h"
+
+ZEND_API void zend_timer_create(void);
+ZEND_API void zend_timer_settime(zend_long seconds);
+ZEND_API void zend_timer_delete(void);
```

Comment:
Something feels off about these functions. They are global, but they don't actually set a timeout handler -- `zend_set_timeout_ex` does that. It's also not clear to me why they are `ZEND_API`.

Reply:
`zend_timer_create()` must be called by extensions and SAPI if they call `fork()` (because `fork()` doesn't copy the timers of the parents). I added the other functions for consistency but it's not necessary. Maybe should I keep only `zend_timer_create()` and rename it `zend_timers_startup()` for consistency with other subsystems.